app/Importer/ItemImporter.php

```diff
@@ -372,7 +372,7 @@ public function createOrFetchStatusLabel($asset_statuslabel_name)
         if (empty($asset_statuslabel_name)) {
             return null;
         }
-        $status = Statuslabel::where(['name' => $asset_statuslabel_name])->first();
+        $status = Statuslabel::where(['name' => trim($asset_statuslabel_name]))->first();
```

Comment:
There's a syntax error on this line. I think this will work:
```suggestion
       $status = Statuslabel::where(['name' => trim($asset_statuslabel_name)])->first();
```